deploy/helm/sumologic/values.yaml

```diff
@@ -1061,7 +1061,9 @@ fluent-bit:
   podLabels: {}
 
   ## Add custom pod annotations to fluent-bit daemonset pods
-  podAnnotations: {}
+  podAnnotations:
+    ## This annotation triggers a restart of the fluent-bit pod on upgrade to helm chart v3
+    sumologic.com/major-version: "3"
```

Comment:
I'm wondering if this is necessary now that we've changed the key in the configmap. In principle it should restart the Fluent-Bit Pods with just that.

Reply:
yes, it looks like the annotation is not necessary, the fluent-bit pod restarted on upgrade,

Reply:
this change is removed.